mappings/net/minecraft/entity/mob/PiglinBrain.mapping

```diff
@@ -137,16 +140,41 @@ CLASS net/minecraft/class_4838 net/minecraft/entity/mob/PiglinBrain
 		ARG 0 entity
 	METHOD method_24791 hasSoulFireNearby (Lnet/minecraft/class_4836;)Z
 		ARG 0 piglin
-	METHOD method_24848 (Lnet/minecraft/class_1542;)Lnet/minecraft/class_1799;
+	METHOD method_24848 getItemFromStack (Lnet/minecraft/class_1542;)Lnet/minecraft/class_1799;
 		ARG 0 stack
-	METHOD method_24849 (Lnet/minecraft/class_4836;Lnet/minecraft/class_1799;)V
+	METHOD method_24849 barterItem (Lnet/minecraft/class_4836;Lnet/minecraft/class_1799;)V
 		ARG 0 piglin
 		ARG 1 stack
 	METHOD method_24850 doesNotHaveGoldInOffHand (Lnet/minecraft/class_4836;)Z
 		ARG 0 piglin
 	METHOD method_24917 hasItemInOffHand (Lnet/minecraft/class_4836;)Z
 		ARG 0 piglin
+	METHOD method_25948 pickupItemWithOffHand (Lnet/minecraft/class_4836;)V
+		ARG 0 piglin
 	METHOD method_26350 getNearbyPiglins (Lnet/minecraft/class_4836;)Ljava/util/List;
 		ARG 0 piglin
+	METHOD method_27086 isWillingToTrade (Lnet/minecraft/class_4836;Lnet/minecraft/class_1799;)Z
+		ARG 0 piglin
+		ARG 1 nearbyItems
+	METHOD method_29277 canRideHoglin (Lnet/minecraft/class_4836;)Z
+		ARG 0 piglin
 	METHOD method_29534 isZombified (Lnet/minecraft/class_1299;)Z
 		ARG 0 entityType
+	METHOD method_29537 getNearestZombifiedPiglin (Lnet/minecraft/class_4836;)Z
+		ARG 0 piglin
+	METHOD method_29538 hasTargetToAvoid (Lnet/minecraft/class_4836;)Z
+		ARG 0 piglin
+	METHOD method_29945 angerNearbyPiglins (Lnet/minecraft/class_4836;)V
+		ARG 0 piglin
+	METHOD method_29946 angerAtPlayer (Lnet/minecraft/class_4836;Lnet/minecraft/class_1309;)V
```

Comment:
This could then become `becomeAngryWithPlayer` and the current `angerAt` become `becomeAngryWith`